README-MS.md

```diff
@@ -134,11 +134,10 @@ Cari pautan yang mengarahkan secara langsung kepada masalah yang sesuai untuk di
 
 ## Sumbangan inisiatif Sumber Terbuka
 - [Up For Grabs](https://up-for-grabs.net/) - Mengandungi isu projek yang mesra penyumbang baru.
-- [First Timers Only](https://www.firsttimersonly.com/) - Senarai pepijat yang dilabel "first-timer-only".
```

Comment:
This can be kept and added back in.

Reply:
done add66b602f87077fb9a28bfc9c24e09203be26b5